CONTRIBUTING.md

```diff
@@ -43,6 +43,41 @@ Additionally, please follow these rules:
 * Use informative but concise variable names. Single-letter names are fine if the context is clear.
 * Avoid explaining code with comments. If something is hard to understand, simplify or decompose it.
 
+### Pre-Commit Hooks
```

Comment:
Could you put the following after the first paragraph of the `# Contribute code` section?

---

To get started with contributing code, we recommend to install `zuko` in [editable mode](https://pip.pypa.io/en/latest/topics/local-project-installs) with its development dependencies.

```
pip install -e .[dev]
```

Optionally, we also provide pre-commit hooks to ensure that the code you commit adheres to our conventions.

```
pre-commit install --config pre-commit.yml
```

After installation, `pre-commit` will automatically execute the configured hooks before each commit and provide instructions on how to fix detected issues.